qiskit_ionq/exceptions.py

```diff
@@ -118,10 +122,11 @@ def from_response(cls, response):
         """
         # TODO: Pending API changes will cleanup this error logic:
         status_code = response.status_code
+        http_response = response.reason
```

Comment:
Reason just returns the text form of the http code, yeah?
https://requests.readthedocs.io/en/latest/api/#requests.Response.reason

I think it's an improvement, but could we also log response headers and the full body? (for example we strip parts of the body if code is found below, and that might be useful for debugging as well)

Reply:
sounds good i included the headers and full body